### PR TITLE
Calculate performance of MTI

### PR DESCRIPTION
**Pull request by nsorros**
Extend script that creates sample to create file in MTI input format.
Add script that reads both files i.e. the sample with the gold tags and the one output that MTI batch service sents back via email and calculates performance for a subset of mesh

### CODE REVIEW

**Comment by nsorros** [comment]
[Calculate accuracy of MTI](https://trello.com/c/I6gengGr/1093-calculate-accuracy-of-mti)

**Comment by nsorros** [comment]
@aCampello and @lizgzil just a reminder about this PR

File: scripts/create_mesh_sample.py [hunk 1]
```diff
@@ -31,14 +31,26 @@ def yield_sample_data(data_path, sample_size=SAMPLE_SIZE, random_seed=RANDOM_SEE
...
+def create_mesh_sample(mesh_data_path, sample_mesh_data_path, mti_input_path):
+    if mti_input_path:
+        mti_f = open(mti_input_path, "w")
+
     with open(sample_mesh_data_path, "w") as f:
-        for item in yield_sample_data(mesh_data_path):
+        for i, item in enumerate(yield_sample_data(mesh_data_path)):
             f.write(item)
 
+            if mti_input_path:
```
Comment (aCampello) [inline scripts/create_mesh_sample.py:42]: Perhaps a comment somewhere with a path to what is expected by MTI and what is output in the e mail? Or maybe in the README, add an example of command line/output for possible reproducibility?  Seems to me that it is as simple as a file with `unique_id|text`.

File: scripts/evaluate_mti.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            item = json.loads(line)
+            gold_tags.append(item["tags"])
+    return gold_tags
+
+def get_mti_tags(mti_output_path, mesh_tags):
+    mti_data = dict()
+    with open(mti_output_path) as f:
+        csvreader = csv.reader(f, delimiter="|")
+        for line in csvreader:
+            uid, tag, _, _ = line
```
Comment (aCampello) [inline scripts/evaluate_mti.py:28]: what is the other two returned fields?

**Review by aCampello — APPROVED** [review]
Looks good. Some very minor suggestions on documentation.

File: scripts/evaluate_mti.py (hunk 2 above)
Comment (nsorros) [reply scripts/evaluate_mti.py:28]: cuid and score. the first is a unique identifier of the tag, score is not a probability but a custom scoring function. why?

File: scripts/create_mesh_sample.py (hunk 1 above)
Comment (nsorros) [reply scripts/create_mesh_sample.py:42]: 76c76dd

File: scripts/evaluate_mti.py (hunk 2 above)
Comment (aCampello) [reply scripts/evaluate_mti.py:28]: I was just curious about why you are not using them. Makes sense.

**Pull request merged**